cornice/service.py

```diff
@@ -413,17 +413,21 @@ def cors_enabled(self):
     def cors_enabled(self, value):
         self._cors_enabled = value
 
-    @property
-    def cors_supported_headers(self):
+    def cors_supported_headers(self, method=None):
```

Comment:
Also, that would let us be backwards compatible, exposing the properties to the people who where already using them. In this case, we should issue a deprecation warning and remove it in a later release.